### ISSUE TITLE
Rendered output displays "See also:" line twice

### ISSUE THREAD

**Issue by kbdharun** [tldr-pages/tldr-node-client#431]
## Expected behavior

Continuing https://github.com/tldr-pages/tldr/issues/5190#issuecomment-1852054873 (cc @sbrl).

The rendered output of pages should be displayed as is without a redundant "See also:" entry below the pages.

## Actual behavior

Currently, the rendered output of pages displays the "See also:" line twice and in pages using a description like `encfs` it might be useful but still redundant.

![image](https://github.com/tldr-pages/tldr-node-client/assets/26346867/34de8701-62ef-496b-9761-ca1c0e1b662f)

Also in pages where we add it to an existing line in those cases this additional rendering doesn't take place:

![image](https://github.com/tldr-pages/tldr-node-client/assets/26346867/e27d715f-3ffe-439e-b586-e41069337dfa)

What do you guys think, should we remove it? (IMO removing it will make the rendering consistent with other clients which doesn't do this)

If we decide to go with it, removing the below lines would resolve this issue:

https://github.com/tldr-pages/tldr-node-client/blob/3e65c255e690ab98bdb9e9e91003eb435a39bc8b/lib/render.js#L48-L52

## Environment

- Operating system - Linux (Ubuntu LTS 22.04, Fedora Workstation 39), Windows (11)
- Node.js version (`node --version`): `v20.9.0`
- tldr-node-client version (`tldr --version`): `3.4.0`

**Comment by sbrl** [comment]
Where is the bottom `See also` pulled from? If it's taken from the page content, I don't particularly see a reason to keep it.

I wonder what the reasoning was behind adding it in the first place was?

**Comment by kbdharun** [comment]
> Where is the bottom `See also` pulled from? If it's taken from the page content, I don't particularly see a reason to keep it.

Yep, it's taken from the page content.

> I wonder what the reasoning was behind adding it in the first place was?

Same here, it was implemented in #81 and it seems to be discussed here in https://github.com/tldr-pages/tldr/issues/481.

**Comment by owenvoke** [comment]
I agree that it doesn't really make much sense, as it's usually already within the page. 👍🏻

**Comment by sbrl** [comment]
Ahh I see. This feature parses

```markdown
(see `blah`)
```

lines in descriptions, but fails to account for if they are mentioned in the `See also` description line.

In this case, I would consider this a bug.
